pyndustri.pyi

```diff
@@ -52,10 +52,118 @@ class Env:
         Underscores become dashes (e.g. `Env.titanium_conveyor` becomes `@titanium-conveyor`).
         """
 
+class World:
+    """
+    World processor specific items
+    """
+
+    @staticmethod
+    def fetch(type: str, team: str, index: int):
+        """Fetches the item at a certain index. Type: {unit|player|core}."""
+    @staticmethod
+    def count(type: str, team: str) -> int:
+        """Get the thing count of a team, where thing: {unit|player|core}."""
+    @staticmethod
+    def fetch_block(type: str, team: str, index: int):
+        """Get the block of type type (eg "Env.conveyor") at index index."""
+    @staticmethod
+    def block_count(type: str, team: str) -> int:
+        """Get the block count of type type (eg "Env.conveyor")."""
+    @staticmethod
+    def get_block(type: str, x: int, y: int):
+        """Get the thing at x, y, type: {floor|ore|block|building}."""
+    @staticmethod
+    def set_block(x: int, y: int, block: str, team: str, rotation: int = 0):
+        """Sets the block at (x, y)."""
```

Comment:
A bit strange that the order of `x, y` does not match. `build` for instance wants `x, y, block_type`, so maybe it's fine to do that here too.

Reply:
What would be cool is if we could do `World[x][y].block(Env.sharded, 0)`

Reply:
I would like to be able to do World[x][y].set(Env.sharded, 0), but how do i put that in the pyndustric.pyi file?

Reply:
Did `set`, next is `get`...